dd-sdk-android-ndk/src/main/cpp/utils/signal-monitor.c

```diff
@@ -18,6 +18,7 @@
 #include <string.h>
 #include <unistd.h>
 
+#include "backtrace-handler.h"
 #include "../datadog-native-lib.h"
```

Comment:
Same comment about the `..`, I'm sure you can tweak the build file to make it work without them